setup.cfg

```diff
@@ -28,11 +28,12 @@ install_requires =
     cloudpickle>=2.2.0,<2.3.0;python_version>='3.11'
     click>=8.1.0
     fastapi
-    grpclib==0.4.3
+    grpclib==0.4.7
     importlib_metadata>=3.6.0
     protobuf>=3.19,<5.0,!=4.24.0
     rich>=12.0.0
-    synchronicity~=0.5.3
+    # TODO revert after testing synchronicity~=0.5.3
+    synchronicity @ git+https://github.com/modal-labs/synchronicity.git@michael/ci-python-versions
```

Comment:
TODO: Land the change in synchronicity, cut a release, then revert this change and bump the synchronicity pin.